okta-idx-kotlin/src/main/java/com/okta/idx/kotlin/client/InteractionCodeFlow.kt

```diff
@@ -131,7 +132,13 @@ class InteractionCodeFlow(val client: OAuth2Client) {
      */
     suspend fun proceed(remediation: IdxRemediation): OAuth2ClientResult<IdxResponse> {
         val request = withContext(client.configuration.computeDispatcher) {
-            remediation.asJsonRequest(client)
+            remediation.asJsonRequest(client).let { req ->
+                if (req.headers["Origin"].isNullOrBlank()) {
+                    req.newBuilder()
+                        .header("Origin", client.configuration.discoveryUrl.toHttpUrlOrNull()?.let { "${it.scheme}://${it.host}" } ?: "")
```

Comment:
Interesting using the discovery URL instead of the issuer URL.  Is there a particular reason?